src/guardian/staking-module-guard/staking-module-guard.service.ts

```diff
@@ -412,6 +425,30 @@ export class StakingModuleGuardService {
     await this.guardianMessageService.sendDepositMessage(depositMessage);
   }
 
+  public async getInvalidKeys(
+    stakingModuleData: StakingModuleData,
+    blockData: BlockData,
+  ): Promise<{ key: string; depositSignature: string }[]> {
+    this.logger.log('Start keys validation', {
+      keysCount: stakingModuleData.vettedKeys.length,
+    });
+    const validationTimeStart = performance.now();
```

Comment:
Let's do performance check with decorators.

Reply:
lets not do))

Reply:
separate business logic and metrics. It will be more easier to test and read this code

Reply:
performance - is a log here

Reply:
and we do the same in other places

Reply:
ok =(